mcs/tools/mkbundle/mkbundle.cs

```diff
@@ -289,6 +300,12 @@ static int Main (string [] args)
 				if (!quiet)
 					Console.WriteLine ("WARNING:\n  Check that the machine.config file you are bundling\n  doesn't contain sensitive information specific to this machine.");
 				break;
+                        case "--no-machine-config":
```

Comment:
nit: indentation

Reply:
No clue what you mean.

Reply:
@migueldeicaza you're using spaces instead of tabs here

Reply:
Got it - done